internal/command/ips/allocate.go

```diff
@@ -70,6 +76,15 @@ func runAllocateIPAddressV4(ctx context.Context) error {
 	if flag.GetBool(ctx, "shared") {
 		addrType = "shared_v4"
 	}
+	if !flag.GetBool(ctx, "yes") {
+		confirmIPV4, err := prompt.Confirm(ctx, "Looks like you're accessing a paid feature. Dedicated IPv4 addresses now costs $2/mo. Are you ok with this?")
+		if err != nil {
```

Comment:
Standard way to check for non-interactive input is `prompt.IsNonInteractive(err)`

Reply:
Oh neat til